bin/setup

```diff
@@ -4,5 +4,6 @@ IFS=$'\n\t'
 set -vx
 
 bundle install
+bundle lock --add-platform x86_64-linux
```

Comment:
Adding this line fix the CI

Reply:
oh nice, what did this do / what was the issue?

Reply:
In the gemfile.lock it set linux 64 bit processor as one of the platforms. It seems like that when you install ruby through rbenv or possibly just ruby in general it defaults to the 64 bit version of Ruby.

Reply:
Looking back at the failed pipeline we can see it trying to use 64 bit version as well:
```
Error: Error: The process '/opt/hostedtoolcache/Ruby/3.1.2/x64/bin/bundle' failed with exit code 16
    at ExecState._setResult (/home/runner/work/_actions/ruby/setup-ruby/v1/dist/index.js:5381:25)
    at ExecState.CheckComplete (/home/runner/work/_actions/ruby/setup-ruby/v1/dist/index.js:5[36](https://github.com/Nauthiz-Jera/ruby-technical/actions/runs/3889361925/jobs/6637556899#step:3:43)4:18)
    at ChildProcess.<anonymous> (/home/runner/work/_actions/ruby/setup-ruby/v1/dist/index.js:5258:27)
    at ChildProcess.emit (node:events:390:28)
    at maybeClose (node:internal/child_process:1064:16)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:301:5)
```

Reply:
Oh nice, looks like you got it working now =]